py3status/modules/spaceapi.py

```diff
@@ -3,26 +3,27 @@
 Display status of a given hackerspace.
 
 Configuration parameters:
-    button_url: Button that when clicked opens the URL sent in the space's API.
-        Setting to None disables. (default 3)
-    cache_timeout: Set timeout between calls in seconds (default 60)
-    closed_text: text if space is closed, strftime parameters
-        will be translated (default 'closed')
-    open_text: text if space is open, strftime parmeters will be translated
-        (default 'open')
-    time_text: format used for time display (default ' since %H:%M')
-    url: URL to SpaceAPI json file of your space
+    button_url: mouse button to open URL sent in space's API (default 3)
+    cache_timeout: refresh interval for this module (default 60)
+    format: display format for this module (default '{spaceapi}')
```

Comment:
I think we should just have a single format eg `{state} since {changed}`  I'd still like a nicer way to do the time format but maybe we cannot.